src/components/FormSections/Addresses.jsx

```diff
@@ -260,6 +266,7 @@ function Addresses(props){
                                         {mailStateField.label}{(addressFieldsState.required === "1") && <span className='required-text'>*</span>}
                                     </Label>
                                         <StateSelector
+                                            data-test="select"
```

Comment:
This is intentionally reused? or does it need to be unique for the tests?

Reply:
I reused this for any dropdown select options so that when reading the tests its consistent!

Reply:
In the future we should expect these to be unique so we can ensure which select list we are interacting with.